report_compassion/models/res_partner.py

```diff
@@ -40,9 +40,16 @@ def get_receipt(self, year):
                 ("last_payment", "<=", end_date),
                 ("state", "=", "paid"),
                 ("product_id.requires_thankyou", "=", True),
+                # invoice from either the partner, the company or the employee
+                # to obtain the same results when tax receipt is computed from companies or employees
                 "|",
+                # invoice from the partner (when self is either an company or an employee)
                 ("partner_id", "=", self.id),
+                "|",
+                # invoice from the company (when self is an employee)
                 ("partner_id.parent_id", "=", self.id),
+                # invoice from the employees (when self is a company)
+                ("partner_id.child_ids", "=", self.id),
```

Comment:
@davidwul It's strange that this had to be added, because it's the opposite of the task description in which you said "tax receipt for companies are correct".